packages/sui-bundler/README.md

```diff
@@ -108,9 +101,9 @@ register({
 })()
 ```
 
-Debes propocionar un handler para cuando se cachea por primera vez contenido y otro para cuando hay nuevo contenido cacheado y tienes que refrescar la página para poder disfrutarlo.
+You should pass a handler in order to handle when it gets cached for the first time the content and another when you get new content and want to handle how to show a notification to the user in order to let him decide if he wants to refresh the page.
```

Comment:
You should pass a handler in order to handle when content gets cached for the first time